index.html

```diff
@@ -2525,6 +2525,14 @@ <h3>Protecting Application Developers</h3>
 application developer population is to be discouraged.
           </li>
           <li>
+Cryptographic library implementers can provide deprecated and experimental
+cryptographic functionality, but are advised to do so in a way where the
+libraries do not enable such functionality unless explicitly requested by the
+developer, such as via a library option, and if enabled, the library produces
+warnings that deprecated or experimental cryptography has been enabled for the
```

Comment:
From an implementation viewpoint it can be difficult to handle "produce warnings" in a way that is useful.  For instance, in a server startup flood of logging it's easy to miss a warning.  It is advisory, so hopefully library authors and users can handle this somehow.